src/components/ModalDialog/ModalDialog.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from "@emotion/styled";
+import { rgba } from "polished";
+import React, { FunctionComponent, useEffect } from "react";
+import tw from "twin.macro";
+import { vars } from "../../styles";
+
+interface ModalDialogProps {
+  className?: string;
+  show: boolean;
+  close: () => void;
+  children?: React.ReactNode;
+}
+
+export const ModalDialogUnStyled: FunctionComponent<ModalDialogProps> = ({
+  className,
+  show,
+  close,
+  children,
+}) => {
+  useEffect(() => {
+    document.body.style.overflow = show ? "hidden" : "visible";
```

Comment:
Is this the right way to show/hide something?

Reply:
ok i check with boon if got better way of making it show/hide